tests/components/utility_meter/test_sensor.py

```diff
@@ -231,6 +231,133 @@ async def test_state(hass: HomeAssistant, yaml_config, config_entry_config) -> N
     assert state is not None
     assert state.state == "unavailable"
 
+    if not yaml_config:
```

Comment:
We should avoid if statements in tests. Please create a own test for it

Reply:
Removed, this is already handled in `test_state_always_available`.